### PR TITLE
Add standalone dev chain directions to README

### PR DESCRIPTION
**Pull request by ferrell-code**
Signed-off-by: Charles Ferrell <charlie@manta.network>

## Description

Adds docs for running dev chain

follow up to #870

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [ ] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

**Review by ghzlatarev — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ This is the mono-repo for Manta/Calamari/Dolphin nodes.
   cargo b --profile production
   ```
 > Tips: The binary will be generated under `target/production/manta`. For less performance critical build, `cargo build --release` is recommended for faster build time.
+3. Run standalone dev chain, useful for testing
+  ```bash
+  cargo run -- --chain=dolphin-localdev --alice --force-authoring --tmp
```
Comment (Garandor) [inline README.md]: iirc we discussed with @zqhxuyuan that force-authoring is not needed

**Review by Garandor — CHANGES_REQUESTED** [review]
would change it myself but i havent tested it works without the flag

File: README.md (hunk 1 above)
Comment (ferrell-code) [reply README.md]: Yep good catch, it doesn't do anything

Comment (ferrell-code) [reply README.md]: Works the same without it

**Review by Garandor — APPROVED** [review]

**Review by ghzlatarev — APPROVED** [review]

**Pull request merged**